app/components/validation-feedback/component.js

```diff
@@ -2,6 +2,22 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   tagName: 'span',
-  classNames: ['input-group-addon']
+  classNames: ['input-group-addon'],
+
+  /** @type {Boolean} Whether errors will show even if no value */
+  hasSubmitted: false,
+
+  shouldShowFeedback: Ember.computed.or("value", "hasSubmitted"),
+
+  errorText: function() {
+    var error = this.get("error");
+
+    // If multiple errors, join errors with space and comma
+    if(Array.isArray(error) && error.join) {
```

Comment:
Ember has an `isArray` that is ES3-safe fyi ([docs](http://emberjs.com/api/#method_isArray)), but we don't support IE8 anyway :-D